### PR TITLE
Add Dockerfile

### PR DESCRIPTION
**Pull request by kafkaca**
I added chromium and firefox with alpine packages. Also i wrote about how build and run of this docker container in readme file.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -25,6 +25,10 @@ references:
     run:
       name: Deploy storybook to AWS
       command: aws s3 sync ./packages/frontend/storybook-static/ s3://components.ucurtmaprojesi.com --delete
+  e2etest_bs: &e2etest_bs
```
Comment (msusur) [inline .circleci/config.yml]: can we use `browserstack` rather than `bs`? I'm concerned about less likely used abbr. :)

File: .circleci/config.yml [hunk 2]
```diff
@@ -25,6 +25,10 @@ references:
...
+    run:
+      name: e2e test process on non-prod.ucurtmaprojesi.com
+      command: yarn e2e:bs
```
Comment (msusur) [inline .circleci/config.yml]: same here, there is no harm using `browserstack`

File: docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,19 @@
+version: "3.1"
+services:
+  ucurtma-container:
+    build:
+      context: ./
+      dockerfile: ./docker/alpine/Dockerfile
+    container_name: ucurtma-container
+    environment:
+      - BROWSERSTACK_USERNAME=0
```
Comment (msusur) [inline docker-compose.yml]: is it possible to use a `.env` file for enforcing the environment variables here?

File: docker-compose.yml [hunk 4]
```diff
@@ -0,0 +1,19 @@
+version: "3.1"
+services:
+  ucurtma-container:
+    build:
+      context: ./
+      dockerfile: ./docker/alpine/Dockerfile
```
Comment (msusur) [inline docker-compose.yml]: folder structure seems quite good, but this `Dockerfile` seems to be customised for testing and doesn't really look like the bare `alpine` image. Can we rename the folder to be more descriptive of what that image is about?

File: package.json [hunk 5]
```diff
@@ -1,37 +1,36 @@
...
-    "url": "git+https://github.com/Ucurtma/ucurtma-app.git"
-  },
-  "bugs": {
-    "url": "https://github.com/Ucurtma/ucurtma-app/issues"
-  },
-  "devDependencies": {
-    "husky": "^2.4.1",
-    "lerna": "^3.15.0"
-  }
+	"name": "ucurtma-app",
```
Comment (msusur) [inline package.json]: As a standard, we use 1 tab indentation (2 spaces). This seems to be off the standard :(

File: packages/frontend/package.json [hunk 6]
```diff
@@ -1,73 +1,70 @@
...
-    "eslint-plugin-prettier": "^3.1.0",
-    "eslint-plugin-react": "7.14.2",
-    "eslint-plugin-react-hooks": "^1.6.1",
-    "gherkin-testcafe": "^2.3.2",
-    "lint-staged": "^8.2.1",
-    "prettier": "^1.18.2",
-    "pretty-quick": "^1.11.1",
-    "testcafe": "^1.3.1"
-  }
+	"name": "ucurtma-frontend",
```
Comment (msusur) [inline packages/frontend/package.json]: 2 spaces 💕

File: packages/frontend/package.json [hunk 7]
```diff
@@ -1,73 +1,70 @@
...
+		"react-dom": "^16.8.6",
+		"react-dropzone": "^10.1.5",
+		"tailwindcss": "^1.0.4",
+		"yup": "^0.27.0"
+	},
+	"resolutions": {
+		"c21e": "1.1.9"
+	},
+	"lint-staged": {
+		"*.{js,jsx}": [ "eslint --ext", "git add" ],
```
Comment (msusur) [inline packages/frontend/package.json]: arrays should break the line :( I started to feel like your ide doesn't respect our prettier config.

File: packages/frontend/tests/google.js [hunk 8]
```diff
@@ -0,0 +1,7 @@
+import { Selector } from 'testcafe';
+
+fixture`Test fixture`.page`${process.env.BASE_URL}`;
+
+test('Take a screenshot google.com', async t => {
```
Comment (msusur) [inline packages/frontend/tests/google.js]: not sure what this test's purpose is? I reckon this needs to be 🔪 :)

File: packages/frontend/tests/features/bs-student-onboarding-journey.feature.txt [hunk 9]
```diff
@@ -0,0 +1,9 @@
+Feature: The student onboarding journey
+
```
Comment (msusur) [inline packages/frontend/tests/features/bs-student-onboarding-journey.feature.txt]: I think the file name shouldn't include the environment or agent-specific descriptions. Ideally, all tests should run on the agent environment that was specified by the command that is running them.

Just noticed that this is a `.txt` file. Shouldn't that be a `.feature`?? Or is this file needed to be deleted along with the `google.js` ?

File: package.json (hunk 5 above)
Comment (kafkaca) [reply package.json]: sorry I couldn't think, I've arranged it now

File: packages/frontend/tests/features/bs-student-onboarding-journey.feature.txt (hunk 9 above)
Comment (kafkaca) [reply packages/frontend/tests/features/bs-student-onboarding-journey.feature.txt]: There was red in the test stage of ci because we couldn't write a test scenario not yet.

**Comment by kafkaca** [comment]
ci variables are not entered, so it gives an error on circleci
